config/initializers/assets.rb

```diff
@@ -5,7 +5,7 @@
 Rails.application.config.assets.precompile += %w( custom/toggle_comment.js custom/toggle_description.js product.css )
 Rails.application.config.assets.precompile += %w( custom.css controllers/hello_controller.js controllers/index.js)
 
```

Comment:
Huynh sửa lại gọn hơn nè
```suggestion
Rails.application.config.assets.precompile += %w( custom/toggle_comment.js custom/toggle_description.js product.css base.scss custom.css controllers/hello_controller.js controllers/index.js)
```